src/components/Calculator.js

```diff
@@ -12,36 +52,36 @@ class Calculator extends React.Component {
             <col span="1" id="col-2" />
           </colgroup>
           <tr>
-            <th colSpan="4"><input type="text" placeholder="0" aria-label="o" /></th>
+            <th colSpan="4"><input type="text" placeholder="0" aria-label="o" value={displays} /></th>
           </tr>
           <tr>
-            <td>AC</td>
-            <td>+/-</td>
-            <td>%</td>
-            <td>/</td>
+            <td><button type="button" name="AC" onClick={this.inputDisplay}>AC</button></td>
+            <td><button type="button" name="+/-" onClick={this.inputDisplay}>+/-</button></td>
+            <td><button type="button" name="%" onClick={this.inputDisplay}>%</button></td>
+            <td><button type="button" name="/" className="orange" onClick={this.inputDisplay}>/</button></td>
```

Comment:
The division operator sign is given an unknown operation error

![image](https://user-images.githubusercontent.com/66112760/148466167-5354c2fc-667c-4f51-a532-0a14f7c68cad.png)

Kindly fix this please 👍